agent/src/main/java/org/openjdk/jmc/agent/Agent.java

```diff
@@ -128,8 +128,14 @@ public static Logger getLogger() {
 	 */
 	private static void initializeAgent(String agentArguments, Instrumentation instrumentation) {
 		if (agentArguments == null || agentArguments.trim().length() == 0) {
-			agentArguments = DEFAULT_CONFIG;
+			try {
+				initializeAgent((InputStream) null, instrumentation);
+			} catch (XMLStreamException e) {
+				// noop
```

Comment:
Nit: perhaps document with something like // noop since null causes defaults to be used - the stream will not be used

Reply:
Thank you. Fixed in b7e05c347f9f634f6d52e9639c619d66c74e9576